src/client.rs

```diff
@@ -59,7 +59,7 @@ pub struct Client<S, T> {
     client_server_map: ClientServerMap,
 
     /// Client parameters, e.g. user, client_encoding, etc.
-    parameters: HashMap<String, String>,
+    _parameters: HashMap<String, String>,
```

Comment:
There are other parameters I'd like us to support long term (client encoding, server encoding, etc.). We'll need this one later.

Reply:
yeah this is just to avoid the warning of unused code

Reply:
```
#[allow(dead_code)]
```